src/flick/friend/serializers.py

```diff
@@ -7,27 +7,16 @@
 from user.models import Profile
 
 
-class FriendProfileSerializer(serializers.ModelSerializer):
-
-    profile_asset_bundle = AssetBundleDetailSerializer(read_only=True)
-
-    class Meta:
-        model = Profile
-        fields = ("bio", "profile_asset_bundle", "owner_lsts", "collab_lsts")
-        read_only_fields = ("id",)
-
-
 class FriendUserSerializer(serializers.ModelSerializer):
-    """
-    User Serializer
-    """
 
-    profile = FriendProfileSerializer(many=False)
+    bio = serializers.CharField(source="profile.bio")
+    profile_pic = AssetBundleDetailSerializer(source="profile.profile_asset_bundle")
```

Comment:
you should distinguish between the different id's -- you can do user_id = serializers.CharField(source='id')